lib/resources/example_json/r4_examples_relatedperson.rb

```diff
@@ -955,6 +1274,35 @@ module Resources
               },
               {
                 'id': 'CI-60187327-0-2',
+                'extension': [
+                  {
+                    'valuePeriod': {
+                      'start': '2016-12-19T16:44:25.000Z',
+                      'end': '2022-05-01T16:56:56.000Z'
+                    },
+                    'url': 'https://fhir-ehr.cerner.com/r4/StructureDefinition/period'
+                  },
+                  {
+                    'valueCodeableconcept': {
+                      'coding': [
+                        {
+                          'system': 'https://fhir.cerner.com/2c400054-42d8-4e74-87b7-80b5bd5fde9f/codeSet/40',
```

Comment:
I don't think mike would be the sister. brother makes more sense for this RelatedPerson

Reply:
changed here https://github.com/cerner/fhir.cerner.com/pull/624/commits/f8e3e624c57363c311b7e4577162e926522db9c8#diff-133ac83b73a8a88b430a777cf1700b9010258bd96bc4ff69a51350091f73643eR414-R415
https://github.com/cerner/fhir.cerner.com/pull/624/commits/f8e3e624c57363c311b7e4577162e926522db9c8#diff-133ac83b73a8a88b430a777cf1700b9010258bd96bc4ff69a51350091f73643eR414-R415

Reply:
changed here f8e3e62#diff-133ac83b73a8a88b430a777cf1700b9010258bd96bc4ff69a51350091f73643eR414-R415
f8e3e62#diff-133ac83b73a8a88b430a777cf1700b9010258bd96bc4ff69a51350091f73643eR414-R415 and other applicable places as well